### PR TITLE
Production deploy #42

### PR DESCRIPTION
**Pull request by despo**
- [x] Bug resolution: Delete Job confirmation popup appearing twice
- [x] Feature: Audit get info button clicks, feedback requests and ToC acceptance.
- [x] Feature: Description meta tag
- [x] Feature: Configure Sidekiq and AWS worker service
- [x] Feature: Add `Suitable for NQT` to vacancy and filters
- [x] Feature: Session timeout increased to 8 hours
- [x] Feature: Update schools rake task configured to run daily
- [x] Move calls to Google index to background jobs
- [x] Fix to terraform variable definition
- [x] Confirm that Support request (3204218) to clear the existing dummy data from the Performance Platform raised by @despo has been resolved. Fiona cc'ed in email. 

### Before merging verify that support request has been resolved.

### CODE REVIEW

File: terraform/modules/ecs/ecs.tf [hunk 1]
```diff
@@ -353,8 +369,42 @@ data "template_file" "logspout_container_definition" {
...
+    aws_elasticsearch_key    = "${var.aws_elasticsearch_key}"
+    aws_elasticsearch_secret = "${var.aws_elasticsearch_secret}"
+
+    rollbar_access_token = "${var.rollbar_access_token}"
+    redis_url            = "${var.redis_url}"
+
+    pp_transactions_by_channel_token = "${var.pp_transactions_by_channel_token}"
+    pp_user_satisfaction_token       = "${var.pp_user_satisfaction_token}"
+    domain                           = "${var.domain}"
+
```
Comment (esteedqueen) [inline terraform/modules/ecs/ecs.tf]: I think we probably want to add the `google_api_json_key` to the env vars here. What do you think?

File: worker_container_definition.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        "value": "${redis_url}"
+      },
+      {
+        "name": "PP_TRANSACTIONS_BY_CHANNEL_TOKEN",
+        "value": "${pp_transactions_by_channel_token}"
+      },
+      {
+        "name": "PP_USER_SATISFACTION_TOKEN",
+        "value": "${pp_user_satisfaction_token}"
+      }
```
Comment (esteedqueen) [inline worker_container_definition.json]: I think we probably want to add `GOOGLE_API_JSON_KEY` and `google_api_json_key` here too.

File: terraform/modules/ecs/ecs.tf (hunk 1 above)
Comment (despo) [reply terraform/modules/ecs/ecs.tf]: 👍

File: worker_container_definition.json (hunk 2 above)
Comment (despo) [reply worker_container_definition.json]: 👍

**Pull request merged**